network/load-balancer/how-to/use-with-private-network.mdx

```diff
@@ -13,7 +13,9 @@ categories:
   - network
 ---
 
-The Private Network feature of the managed Load Balancers product lets you configure your Load Balancer with a Private Network. [Scaleway Private Networks](/network/vpc/concepts/#private-networks) allow your cloud resources to communicate in an isolated and secure network. Load Balancers using Private Networks can communicate with your backend servers without using the public Internet. You can add your Load Balancer to one or multiple existing Private Networks. The feature supports both static IP and DHCP configuration for the Load Balancer IP in the Private Network.
+The Private Network feature of the managed Load Balancers product lets you attach your Load Balancer to up to eight different Private Networks. [Scaleway Private Networks](/network/vpc/concepts/#private-networks) allow your cloud resources to communicate in an isolated and secure network. Load Balancers using Private Networks can communicate with their backend servers without using the public Internet. The feature supports both static IP and DHCP configuration for the Load Balancer IP in each Private Network.
+
+[Private Load Balancers](/network/load-balancer/concepts/#private-load-balancer) must be connected to a Private Network in order to be functional, as they do not have a public IP address and can only receive requests from other resources on the Private Network. This is in contrast to public Load Balancers, which are accessible for requests via the internet through their public IP address, but can optionally use Private Networks to communicate with their backend servers. The procedure for attaching either a public or a private Load Balancer to a Private Network is the same, and described below.
```

Comment:
> The procedure for attaching either a public or a private Load Balancer to a Private Network is the same

@RoRoJ this is not the case as private LB won't offer any attachment mode/option (it will be DHCP only and we won't be showing this in the Console I believe). Public LB will very soon follow with this change in Console (we will remove the static IP address attachment option from Console but keep it in the API for compatibility/breaking changes reasons).

Reply:
OK, I've modified it to make it vague for now